### DISCUSSION TITLE
could not boot :Error 0x7f048281

### DISCUSSION THREAD

**Discussion by zhaoshuyi12** [ipxe/ipxe#625] [Q&A]
When I install windows 10 , I encountered a mistake：

....patching WIM peSetup.cmd at [0x1ad148ee,0x1ad157d8]
Resource exceeds length of file 
Could not patch WIM boot.wim
Could not boot:Error 0x7f048281(https://ipxe.org/7f048281)
Could not boot:Error 0x7f048281 (https://ipxe.org/7f048281)

![微信图片编辑_20220326182211](https://user-images.githubusercontent.com/91310384/160235329-70a44123-0a2e-4c8b-821a-2e7767259b91.jpg)

**Comment by NiKiZe** [comment]
Could not boot is from iPXE when wimboot exits with error.
I recommend that you use the latest version of wimboot.
And that you run imgfree before starting to download files needed for wimboot, as you can see there is other files that is being added to boot.wim that you probably don't need. Check the filesizes with imgstat.

**Reply by NiKiZe** [reply]
https://github.com/ipxe/ipxe/discussions/625#discussioncomment-2448719 says 2.6.0 is used
Latest version is https://github.com/ipxe/wimboot/tree/v2.7.4-signed

**Comment by zhaoshuyi12** [comment]
sorry！ I don't understand what you means .Before,I through this wim boot installed system.I only&nbsp;Change another Linux server to install windows, but wimboot is the same



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Christian ***@***.***&gt;; 
发送时间: 2022年3月26日(星期六) 晚上7:24
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ipxe/ipxe] could not boot :Error 0x7f048281 (Discussion #625)






wim boot 退出时出错，无法从 i PXE 引导。
我建议你使用最新版本的 wim boot 。
在开始下载 wim boot 所需的文件之前，您需要运行 img free ，正如您所看到的，还有其他一些文件正在添加到 boot . wim 中，而这些文件可能并不需要。使用 imgs tat 检查文件大小。

—
直接回复此邮件，在 GitHub 上查看或取消订阅.
您收到这个消息是因为您编写了这个线程。留言 ID ***@***.***与&gt;

**Reply by stappersg** [reply]
Acknowledge on _I don't understand what you mean_.
What follows is a translation (attempt?)
> Could not boot is from iPXE when wimboot exits with error.

wimboot was started and did `exit` with an error,  iPXE is reporting that as _"Could not boot"_

> I recommend that you use the latest version of wimboot.

Tell us which version of wimboot is being used.

> And that you run imgfree before starting to download files needed for wimboot, as you can see there is other files that is being added to boot.wim that you probably don't need.

Modify your configuration, insert somewhere `imgfree`  (note from translator: I don't know where)

> Check the filesizes with imgstat.

Press `s` to enter _iPXE shell_,  execute `imgstat` command and compare the size with the size of the files that have been fetched.

**Comment by zhaoshuyi12** [comment]
I used wimboot is&nbsp;wimboot-2.6.0-signed




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Geert ***@***.***&gt;; 
发送时间: 2022年3月26日(星期六) 晚上8:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ipxe/ipxe] could not boot :Error 0x7f048281 (Discussion #625)





 
Acknowledge on I don't understand what you mean.
 What follows is a translation (attempt?)
  
Could not boot is from iPXE when wimboot exits with error.
  
wimboot was started and did exit with an error,  iPXE is reporting that as "Could not boot"
  
I recommend that you use the latest version of wimboot.
  
Tell us which version of wimboot is being used.
  
And that you run imgfree before starting to download files needed for wimboot, as you can see there is other files that is being added to boot.wim that you probably don't need.
  
Modify your configuration, insert somewhere imgfree  (note from translator: I don't know where)
  
Check the filesizes with imgstat.
  
Press s to enter iPXE shell,  execute imgstat command and compare the size with the size of the files that have been fetched.

**Reply by NiKiZe** [reply]
Old version!

Latest is https://github.com/ipxe/wimboot/tree/v2.7.4-signed

Use latest version of wimboot

**Comment by zhaoshuyi12** [comment]
I couldn't enter iPXE shell . When I&nbsp; press s ,reboot again




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Geert ***@***.***&gt;; 
发送时间: 2022年3月26日(星期六) 晚上8:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ipxe/ipxe] could not boot :Error 0x7f048281 (Discussion #625)





 
Acknowledge on I don't understand what you mean.
 What follows is a translation (attempt?)
  
Could not boot is from iPXE when wimboot exits with error.
  
wimboot was started and did exit with an error,  iPXE is reporting that as "Could not boot"
  
I recommend that you use the latest version of wimboot.
  
Tell us which version of wimboot is being used.
  
And that you run imgfree before starting to download files needed for wimboot, as you can see there is other files that is being added to boot.wim that you probably don't need.
  
Modify your configuration, insert somewhere imgfree  (note from translator: I don't know where)
  
Check the filesizes with imgstat.
  
Press s to enter iPXE shell,  execute imgstat command and compare the size with the size of the files that have been fetched.

**Comment by zhaoshuyi12** [comment]
I enter S enter ipxe shell . imgstat I compare the size with the size of the files that have been fetched are same. I download wimboot、peSetup.cmd、bcd、boot.sdi、boot.wim




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Geert ***@***.***&gt;; 
发送时间: 2022年3月26日(星期六) 晚上8:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ipxe/ipxe] could not boot :Error 0x7f048281 (Discussion #625)





 
Acknowledge on I don't understand what you mean.
 What follows is a translation (attempt?)
  
Could not boot is from iPXE when wimboot exits with error.
  
wimboot was started and did exit with an error,  iPXE is reporting that as "Could not boot"
  
I recommend that you use the latest version of wimboot.
  
Tell us which version of wimboot is being used.
  
And that you run imgfree before starting to download files needed for wimboot, as you can see there is other files that is being added to boot.wim that you probably don't need.
  
Modify your configuration, insert somewhere imgfree  (note from translator: I don't know where)
  
Check the filesizes with imgstat.
  
Press s to enter iPXE shell,  execute imgstat command and compare the size with the size of the files that have been fetched.

**Comment by stappersg** [comment]
On Sun, Mar 27, 2022 at 07:36:34PM -0700, zhaoshuyi12 wrote:
> I enter S enter ipxe shell . imgstat I compare the size with the
> size of the files that have been fetched are same. I download
> wimboot、peSetup.cmd、bcd、boot.sdi、boot.wim

What is the output of `imgstat`?

What is the output of `ls -l` in the directory at the server
where those files reside?  Yes, I asking for the output of a Linux
command, it might need translation.
I don't care about MS Windows, but do care about iPXE.

**Comment by zhaoshuyi12** [comment]
??? ls -l ? ipxe command not found .I don't know what you said






------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Geert ***@***.***&gt;; 
发送时间: 2022年3月28日(星期一) 下午2:24
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ipxe/ipxe] could not boot :Error 0x7f048281 (Discussion #625)





 On Sun, Mar 27, 2022 at 07:36:34PM -0700, zhaoshuyi12 wrote:
 &gt; I enter S enter ipxe shell . imgstat I compare the size with the
 &gt; size of the files that have been fetched are same. I download
 &gt; wimboot、peSetup.cmd、bcd、boot.sdi、boot.wim
 
 What is the output of `imgstat`?
 
 What is the output of `ls -l` in the directory at the server
 where those files reside?  Yes, I asking for the output of a Linux
 command, it might need translation.
 I don't care about MS Windows, but do care about iPXE.

**Reply by stappersg** [reply]
Let me write it again:
```
  Yes, I asking for the output of a Linux command, it might need translation.
  I don't care about MS Windows, but do care about iPXE.
```     

And because I care about iPXE:

* What is the output of `imgstat`
* What is the output of `dir`  in the directory at the server where the to be downloaded files reside?

**Reply by zhaoshuyi12** [reply]
> Let me write it again:
> 
> ```
>   Yes, I asking for the output of a Linux command, it might need translation.
>   I don't care about MS Windows, but do care about iPXE.
> ```
> 
> And because I care about iPXE:
> 
> * What is the output of `imgstat`
> * What is the output of `dir`  in the directory at the server where the to be downloaded files reside?
![微信图片编辑_20220328153954](https://user-images.githubusercontent.com/91310384/160350522-69326c9d-4508-458b-89eb-631c71caca86.jpg)

**Reply by stappersg** [reply]
* What is the output of `dir`  in the directory at the server where the to be downloaded files reside?

**Reply by zhaoshuyi12** [reply]
> 
![微信图片_20220328155915](https://user-images.githubusercontent.com/91310384/160353044-c3552bad-bc26-4e83-abe8-1857eea1edc0.jpg)

**Reply by zhaoshuyi12** [reply]
> * What is the output of `dir`  in the directory at the server where the to be downloaded files reside?

Did you see my reply

**Comment by zhaoshuyi12** [comment]
look at the picture:






------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Geert ***@***.***&gt;; 
发送时间: 2022年3月28日(星期一) 下午3:35
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ipxe/ipxe] could not boot :Error 0x7f048281 (Discussion #625)





 
Let me write it again:
   Yes, I asking for the output of a Linux command, it might need translation.   I don't care about MS Windows, but do care about iPXE.  
And because I care about iPXE:
  
What is the output of imgstat
 
What is the output of dir  in the directory at the server where the to be downloaded files reside?

**Comment by zhaoshuyi12** [comment]
I change the last version wimboot,but the problem is&nbsp; still existed.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Christian ***@***.***&gt;; 
发送时间: 2022年3月28日(星期一) 晚上11:46
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ipxe/ipxe] could not boot :Error 0x7f048281 (Discussion #625)





 
#625 (comment) says 2.6.0 is used
 Latest version is https://github.com/ipxe/wimboot/tree/v2.7.4-signed

**Reply by stappersg** [reply]
Acknowledge on _I'm still stuck_.
Advice:
* Understand http://www.catb.org/~esr/faqs/smart-questions.html
* Gather further information
* Share that information
